Marlin/src/lcd/dogm/u8g_dev_tft_320x240_upscale_from_128x64.cpp

```diff
@@ -650,7 +650,7 @@ uint8_t u8g_dev_tft_320x240_upscale_from_128x64_fn(u8g_t *u8g, u8g_dev_t *dev, u
           lcd_id = 0;
           return 0;
         default:
-          setWindow = (lcd_id & 0xFF000000) ? setWindow_st7789v : setWindow_ili9328;
+          setWindow = (lcd_id && 0xFF000000) ? setWindow_st7789v : setWindow_ili9328;
```

Comment:
Sure it's right? `something && 0xFF000000` is same thing than testing just `something` since 0xFF000000 is always true

Reply:
If warning is about sign/unsigned maybe a (lcd_id & 0xFF000000U) may fix

Reply:
The original marlin code uses &&. A few days ago, @thinkyhead saw that `&&` and changed to `&`. We all thought it won't be a problem.
But seems some displays are working because that if were __always true__ .. heheh.. So, now, for a test, I just put back the *original code*...
If it work for the user, I will try to find the LCD_ID of his display to properly handle in the switch.

Reply:
If it work this way then you may simply write:
`setWindow = lcd_id ? setWindow_st7789v : setWindow_ili9328;`
or
`setWindow = (lcd_id != 0) ? setWindow_st7789v : setWindow_ili9328;`
if some warning appears

Reply:
Thanks. But I know how to do a `if`. As I said: I just put back the original code.

Reply:
I'm sorry was not to offend you, just that what I proposed remove any doubt about the code. Everyone reading such code may think that && is wrong and change it with a &, as @thinkyhead did

Reply:
No problem. That won’t be the final code. It’s just for the user test. I went back to the original code to be easy to reasoning about the problem, as he said that worked before.

All points the we must remove the if... because if the “always true branch” is correct, we can keep only that path... 

Don’t worry. That abomination won’t be merged that way.